phpunit/functional/LocationTest.php

```diff
@@ -253,6 +253,31 @@ public function testImport(array $input, array $imported): void
         }
     }
 
+    public function testImportSeparator(): void
+    {
+        $instance = new \Location();
+        $imported_id = $instance->import(
+            Sanitizer::sanitize(
+                [
+                    'entities_id'   => 0,
+                    'name'          => '_location01 > _sublocation01',
+                ]
+            )
+        );
+        $this->assertEquals(getItemByTypeName(\Location::class, '_sublocation01', true), $imported_id);
+
```

Comment:
To be extra safe, shouldn't this test also check the created location has the correct parent ?